src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAnAotApp.cs

```diff
@@ -752,6 +752,54 @@ public void NativeAotStaticLib_only_runs_when_switch_is_enabled(string targetFra
             IsNativeImage(publishedDll).Should().BeTrue();
         }
 
+        [Theory]
+        [InlineData(ToolsetInfo.CurrentTargetFramework)]
+        public void NativeAotStaticLib_errors_out_when_eventpipe_is_enabled(string targetFramework)
+        {
+            // Revisit once the issue is fixed
+            // https://github.com/dotnet/runtime/issues/89346
+            var projectName = "AotStaticLibraryPublishWithEventPipe";
+            var rid = EnvironmentInfo.GetCompatibleRid(targetFramework);
+
+            var testProject = CreateTestProjectWithAotLibrary(targetFramework, projectName);
+            testProject.AdditionalProperties["PublishAot"] = "true";
+            testProject.AdditionalProperties["RuntimeIdentifier"] = rid;
+            testProject.AdditionalProperties["NativeLib"] = "Static";
```

Comment:
Is this the only difference between the two tests? In that case you can just add a parameter to the theory.